### PR TITLE
Updated label in cronjob job spec to allow it to be uniquely identifi…

### PR DESCRIPTION
**Pull request by aaronfern**
**How to categorize this PR?**

/area backup
/kind bug

**What this PR does / why we need it**:
Updated label in cronjob job template spec so that it can be identified

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator
NONE
```

### CODE REVIEW

File: charts/etcd/templates/etcd-compaction-cronjob.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ metadata:
     name: {{ .Values.name }}
     uid: {{ .Values.uid }}
 spec:
-  schedule: {{ .Values.backup.backupCompactionSchedule }} 
+  schedule: "{{ .Values.backup.backupCompactionSchedule }}"
```
Comment (amshuman-kr) [inline charts/etcd/templates/etcd-compaction-cronjob.yaml:26]:
```suggestion
  schedule: {{ .Values.backup.backupCompactionSchedule | quote }} 
```

File: charts/etcd/templates/etcd-compaction-cronjob.yaml [hunk 2]
```diff
@@ -41,7 +41,7 @@ spec:
 {{ toYaml .Values.annotations | indent 12 }}
 {{- end }}
           labels:
-            name: etcd
+            name: etcd-compaction-cronjob
```
Comment (amshuman-kr) [inline charts/etcd/templates/etcd-compaction-cronjob.yaml:44]:
```suggestion
            name: etcd-backup-compaction
```
Let's use the `name` label to indicate what process is actually running in the pod. Technically, it is not a cronjob, just a pod. Pedantic, I know, but still 😅

**Review by amshuman-kr — CHANGES_REQUESTED** [review]
Thanks for the quick change @aaronfern! Looks good. Just a couple of pedantic suggestions 🙂

**Review by amshuman-kr — APPROVED** [review]
LGTM!

**Comment by timuthy** [comment]
@aaronfern can this be closed as cronjobs won't be used anymore?

**Comment by aaronfern** [comment]
Yes, this isn't needed anymore. Closing 
/close

**Pull request closed without merging**